docs/tutorials/storage.rst

```diff
@@ -8,7 +8,14 @@ switch the device off and on again.
 
 Happily MicroPython on the micro:bit allows you to do this with a very simple
 file system. Because of memory constraints **there is approximately 30k of
-storage available** on the file system.
+storage available** on the file system. 
+
+.. note::
+
+The micropython file system should not be confused
```

Comment:
The text below `.. note::` needs to indented for it to be styled in the HTML output, otherwise it appears as a normal paragraph.